File: HomeSmart/HomeSmart/Model/LivingRoom.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeSmart.Model
{
    public class LivingRoom
    {
        public string _id { get; set; }
        public string homeId { get; set; }
        public Boolean lamp { get; set; }
        public string airConditioningId { get; set; }
        public DateTime createdAt { get; set; }
        public DateTime updatedAt { get; set; }
    }
}

File: Server/models/LivingRoom.js
const mongoose = require('mongoose');

const LivingRoomSchema = new mongoose.Schema({
    homeId:{
        type: String,
        required: true
    },
    lamp:{
        type: Boolean,
        default: false,
        // false: Tắt, True: Bật
    },
    airConditioningId:{
        type:String,
        default: ""
    }
},{timestamps: true}
)

module.exports = mongoose.model("LivingRoom", LivingRoomSchema);